<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QWidget" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>211</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lab_text">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>330</y>
         <width>311</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(112, 112, 112);</string>
       </property>
       <property name="text">
        <string>密码8-20位,至少包含字母/数字/字符2种组合</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_back">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>2</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> background-color: transparent; 
text-align : left;
</string>
       </property>
       <property name="text">
        <string>&lt;返回登录</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_pwd">
       <property name="geometry">
        <rect>
         <x>102</x>
         <y>288</y>
         <width>211</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_phone">
       <property name="geometry">
        <rect>
         <x>162</x>
         <y>247</y>
         <width>151</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>290</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/lock.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_signin">
       <property name="geometry">
        <rect>
         <x>52</x>
         <y>358</y>
         <width>261</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ea4848;</string>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_countrieslist">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>250</y>
         <width>61</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-repeat: no-repeat;
font: 10pt &quot;Consolas&quot;;
text-align : left;
/*或者通过水平和垂直对齐方式来控制*/
padding-left:2px;</string>
       </property>
       <property name="text">
        <string>+8645</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/images/btn_down_h.png</normaloff>:/images/btn_down_h.png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="lab">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>250</y>
         <width>31</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/phone.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QPushButton" name="btn_back_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>8</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> background-color: transparent; 
text-align : left;
</string>
       </property>
       <property name="text">
        <string>&lt;返回登录</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_captcha">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>270</y>
         <width>141</width>
         <height>35</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>234</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>为了安全,我们会向您的手机发送短信验证码</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_regain">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>270</y>
         <width>81</width>
         <height>35</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ea4848;</string>
       </property>
       <property name="text">
        <string>重新获取</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_next">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>340</y>
         <width>231</width>
         <height>35</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ea4848;</string>
       </property>
       <property name="text">
        <string>下一步</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_error_tip">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>310</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 85, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <zorder>label_2</zorder>
      <zorder>btn_back_2</zorder>
      <zorder>line_captcha</zorder>
      <zorder>label_4</zorder>
      <zorder>btn_regain</zorder>
      <zorder>btn_next</zorder>
      <zorder>lab_error_tip</zorder>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
